docs/pages/tutorials/python.rst

```diff
@@ -1,3 +1,142 @@
+
+Binary files
+^^^^^^^^^^^^
+Maps can be saved to disk using
+
+.. literalinclude:: ../../../examples/python/io/save_map_to_file.py
+    :language: python
+
+.. _python-code-examples-read-map:
+
+and read with
+
+.. literalinclude:: ../../../examples/python/io/load_map_from_file.py
+    :language: python
+
+Configs
+-------
+In the previous mapping pipeline example, the configuration parameters for the map and the measurement integration components were hard-coded. To make your setup more flexible, you can use configuration files. We will demonstrate how to work with YAML files, which is the format we use for wavemap's :gh_file:`example configs <interfaces/ros1/wavemap_ros/config>`. However, pywavemap is flexible and can support any parameter format that can be read into a Python `dict`.
+
+
+YAML files
+^^^^^^^^^^
+
+.. literalinclude:: ../../../examples/python/io/load_params_from_file.py
+    :language: python
+
+
+Queries
+=======
+In this section, we show how you can query wavemap maps and classify whether a point or region of interest is occupied.
+
+Node indices
+------------
+The map models the environment by filling it with cubes of variable sizes, arranged as the nodes of an octree. Node indices are defined as integer [X, Y, Z, height] coordinates, whose XYZ values correspond to the node's position in the octree's grid at the given *height*, or level in the tree. Height 0 corresponds to the map's maximum resolution, and the grid resolution is halved for each subsequent height level.
+
+Fixed resolution
+^^^^^^^^^^^^^^^^
+Querying the value of a single node in the highest resolution grid (*height=0*) can be done as follows.
+
+.. literalinclude:: ../../../examples/python/queries/fixed_resolution.py
+    :language: python
+
+Multi-res averages
+^^^^^^^^^^^^^^^^^^
+It is also possible to query lower resolution nodes, whose values correspond to the average estimated occupancy of the volume they cover.
+
+.. literalinclude:: ../../../examples/python/queries/multi_resolution.py
+    :language: python
+
+Accelerators
+^^^^^^^^^^^^
+If you need to look up multiple node values, we recommend using our batched query functions. These functions deliver significant speedups by utilizing wavemap's QueryAccelerator.
+
+.. literalinclude:: ../../../examples/python/queries/accelerated_queries.py
+    :language: python
+
+.. note::
+
+    So far batched queries are only implemented for HashedWaveletOctree maps. We will add support for HashedChunkedWaveletOctree maps in the near future.
+
+.. _python-code-examples-interpolation:
+
+Real coordinates
+----------------
+Many applications require occupancy estimates at arbitrary 3D points, with real-valued coordinates. Such estimates are computed by interpolating the map.
```

Comment:
If you change the C++ bit to use "3D world coordinates" this should mirror the change.